Bottom: da0c5ff5c4e3c9e4460baebe8b142e53bb181119
Top:    27db1321fde89455afc87de24c9a9f468b59b08c
Author: Jason Wang <jasowang@redhat.com>
Date:   2017-03-20 18:12:07 +0800

Refresh of ptr_ring-introduce-batch

---

diff --git a/include/linux/ptr_ring.h b/include/linux/ptr_ring.h
index 47b0843..4771ded 100644
--- a/include/linux/ptr_ring.h
+++ b/include/linux/ptr_ring.h
@@ -314,7 +314,7 @@ static inline void *ptr_ring_consume_bh(struct ptr_ring *r)
 }
 
 static inline int ptr_ring_consume_batched(struct ptr_ring *r,
-                                           void **array, int n)
+					   void **array, int n)
 {
 	int ret;
 
@@ -326,7 +326,7 @@ static inline int ptr_ring_consume_batched(struct ptr_ring *r,
 }
 
 static inline int ptr_ring_consume_batched_irq(struct ptr_ring *r,
-                                               void **array, int n)
+					       void **array, int n)
 {
 	int ret;
 
@@ -351,7 +351,7 @@ static inline int ptr_ring_consume_batched_any(struct ptr_ring *r,
 }
 
 static inline int ptr_ring_consume_batched_bh(struct ptr_ring *r,
-                                              void **array, int n)
+					      void **array, int n)
 {
 	int ret;
